packages/material-ui/src/TablePagination/TablePagination.test.js

```diff
@@ -149,15 +144,15 @@ describe('<TablePagination />', () => {
         </table>,
       );
 
-      const backButton = wrapper.find(IconButton).at(0);
-      const nextButton = wrapper.find(IconButton).at(1);
-      expect(backButton.props().disabled).to.equal(false);
-      expect(nextButton.props().disabled).to.equal(true);
+      const backButton = getByLabelText('Previous page');
+      const nextButton = getByLabelText('Next page');
+      expect(backButton.disabled).to.equal(false);
```

Comment:
With 
```suggestion
      expect(backButton).to.have.property('disabled', false);
```
you get a more helpful error message if it fails. You'd get something like
`Expected {} to have property 'disabled' equal to 'false' but got 'true'` instead of the non-descriptive `Expected 'true' to equal 'false'`

Reply:
Yes you're right